src/coreComponents/finiteVolume/mimeticInnerProducts/BdVLMInnerProduct.hpp

```diff
@@ -127,7 +127,7 @@ BdVLMInnerProduct::Compute( arrayView2d< real64 const, nodes::REFERENCE_POSITION
     // in the implementation of the transmissibility multiplier (see below)
     // TODO: see what it would take to bring the (harmonically averaged) two-point trans here
     real64 diagEntry = 0.0;
-    MimeticInnerProductHelpers::ComputeInvTPFATransWithMultiplier< NF >( elemPerm,
+    MimeticInnerProductHelpers::computeInvTpfaTransWithMultiplier< NF >( elemPerm,
```

Comment:
`Tpfa` is another one like `Pvt`, but I suggest we leave as it.  It is still very legible.

Reply:
I've put back `TPFA` while surprisingly `Tpfa` is more readable imho. (because you know when to stop reading the letters and start again standard reaging. Same for `PVT`